src/sonic-yang-models/yang-events/sonic-events-bgp.yang

```diff
@@ -0,0 +1,97 @@
+module sonic-events-bgp {
+    namespace "http://github.com/Azure/sonic-events-bgp";
+
+    yang-version 1.1;
+
+    import sonic-events-common {
+        prefix evtcmn;
+    }
+
+    import openconfig-alarm-types {
+        prefix oc-alarm-types;
+    }
+
+    revision 2022-12-01 {
+        description "BGP alert events.";
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC BGP events";
+
+    container bgp-state {
+        oc-alarm-types:MINOR
+
+        description "
+            Declares an event for BGP state for a neighbor IP going up/down.;
+
+        leaf ip {
+            type inet:ip-address;
+            description "IP of neighbor";
+        }
+
+        leaf status {
+            type enumeration {
+                enum "up";
+                enum "down";
+
+            }
+            description "Provides the status as up (true) or down (false)";
+        }
+
+        uses evtcmn:sonic-events-cmn;
+    }
+
+    container notification {
+        oc-alarm-types:MAJOR
+
+        description "
+            Reports an notification.
+            The error codes as per IANA.
+            The other params are as in the message";
+
+        leaf major-code {
+            type uint8;
+            description "Major IANA error code; [RFC4271][RFC7313]";
+        }
+
+        leaf minor-code {
+            type uint8;
+            description "Minor IANA error code; [RFC4271][RFC7313]";
+        }
+
+        leaf ip {
+            type inet:ip-address;
+            description "IP of neighbor associated with this notification";
+        }
+
+        leaf ASN {
+            type uint32;
+            description "ASN number from the notification";
+        }
+
+        leaf is-sent {
+            type boolean;
+            description "true - if this notification was for sent messages; false if it was for received.";
```

Comment:
For my understanding,  can you please refer me to HLD section, which describes is-sent. I'm not able to understand it.